src/Uno.UI/UI/Xaml/UIElement.skia.cs

```diff
@@ -173,7 +173,19 @@ internal void AddChild(UIElement child, int? index = null)
 
 		internal void MoveChildTo(int oldIndex, int newIndex)
 		{
-			ApiInformation.TryRaiseNotImplemented("UIElement", "MoveChildTo");
+			var view = _children[oldIndex];
+
+			_children.RemoveAt(oldIndex);
+			if (newIndex == _children.Count)
+			{
+				_children.Add(view);
+			}
+			else
+			{
+				_children.Insert(newIndex, view);
+			}
+
+			InvalidateMeasure();
```

Comment:
Can we have tests for this where:

1) `oldIndex` == `newIndex`
2) `oldIndex` > `newIndex`
3) `newIndex` > `oldIndex`

Also, on Wasm it seems like we only call `AddView`, would that cause the view to be duplicated? Or does it magically work somehow?

Reply:
Done!

Reply:
Validated on WASM, the behavior is correct there as well